handler/core/src/test/java/io/knotx/fragments/handler/debug/FragmentsDebugModeDecoratorDecoratorTest.java

```diff
@@ -150,4 +148,24 @@ void expectFragmentWithBodyEndTagContainsDebugDataAndScripts() throws IOExceptio
     assertTrue(expectedBody.contains(debugCss));
     assertTrue(expectedBody.contains(debugJs));
   }
+
+  @Test
+  @DisplayName("Expect fragment with debug data for Json Object")
+  void expectFragmentWithDebugDataForJson(){
```

Comment:
Could you please add 2 more tests:
- one that will cover `DEFAULT` decorator
- one that will cover `json` type with `JsonArray` as body case?

Reply:
done